src/xml/ConfigParser.cpp

```diff
@@ -210,7 +210,7 @@ auto gatherCandidates(const std::vector<std::shared_ptr<XMLTag>> &DefTags, std::
 
 void ConfigParser::connectTags(const ConfigurationContext &context, std::vector<std::shared_ptr<XMLTag>> &DefTags, CTagPtrVec &SubTags)
 {
-  std::unordered_set<std::string> usedTags;
+  std::unordered_set<std::string_view> usedTags;
```

Comment:
This works only under the assumption that the tags don't change. Let's keep this `std::string` to make this more robust to future changes.